database.sql

```diff
@@ -0,0 +1,6 @@
+create table github_discord_url(
+    id uuid DEFAULT uuid_generate_v4() not null,
+    created_at timestamptz DEFAULT Now() not null,
+    updated_at timestamptz DEFAULT Now() not null,
+    discord_urls varchar(255) not null UNIQUE
```

Comment:
why is this plural? 
had left this comment before also

Reply:
Oops, I made it singular, but I think so it was in some other branch 🥲